### PR TITLE
Fix the problem that the black and white list string is spliced upside down

### PR DESCRIPTION
**Pull request by BurningCN**

### CODE REVIEW

**Comment by haoyann** [comment]
I tested this feature and it is normal.

File: dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java [hunk 1]
```diff
@@ -251,7 +251,7 @@ public static RoutingRule insertConditionRule(RoutingRule existRule, ConditionRo
         List<String> conditions = new ArrayList<>();
         if (whiteList != null && whiteList.size() > 0) {
             StringBuilder sb = new StringBuilder();
-            sb.append("host != ");
+            sb.append("host = ");
```
Comment (haoyann) [inline dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:254]: I tested this feature and it is normal. This is the document https://dubbo.apache.org/zh/docs/v2.7/user/examples/routing-rule/ , black and white uses `=>` the previous config to filter consumer.

Comment (BurningCN) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:254]: Thanks for your review.
When I configure, the whitelist is `1.1.1.1 `and the blacklist is `2.2.2.2`. The following screenshot is generated. According to my understanding, shouldn't it be like this? `host=1.1.1.1=> and host!=2.2.2.2 =>`, But what is shown below is `host!=1.1.1.1=> and host=2.2.2.2 =>`


image

image

Comment (haoyann) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:254]: This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
image

Comment (BurningCN) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:254]: > This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
> image

Looking at my screenshot, I understand the black and white list of `consumers (not providers)` configured here, right? I configured the consumer whitelist to be 1.1.1.1, but the generated host!=1.1.1.1, isn't the allowed 1.1.1.1 filtered out?

Comment (BurningCN) [reply dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/RouteUtils.java:254]: > This rule provider list is empty. When consumer IP mathch this rule provider list is empty . so it's blacklist.
> image

Thank you very much, according to your description, I have understood

**Pull request closed without merging**